examples/marbot/main.tf

```diff
@@ -0,0 +1,34 @@
+variable "marbot_endpoint_id" {
+  default = ""
```

Comment:
Add description

Reply:
Describe what is marbot.
Looks like it's this https://marbot.io/